devito/core/intel.py

```diff
@@ -0,0 +1,202 @@
+from functools import partial
+
+from devito.core.cpu import CPU64Operator, CPU64OpenMPOperator
+from devito.exceptions import InvalidOperator
+from devito.passes.equations import collect_derivatives
+from devito.passes.clusters import (Blocking, Lift, cire, cse, eliminate_arrays,
+                                    extract_increments, factorize, fuse, optimize_pows)
+from devito.passes.iet import (DataManager, Ompizer, avoid_denormals, mpiize,
+                               optimize_halospots, hoist_prodders, relax_incr_dimensions)
+from devito.tools import as_tuple, timed_pass
+
+__all__ = ['Intel64Operator', 'Intel64OpenMPOperator', 'Intel64FSGOperator',
+           'Intel64FSGOpenMPOperator']
+
+
+Intel64Operator = CPU64Operator
```

Comment:
necessary?

Reply:
Yes, needed for registry...though it can be removed, but is better like that IMHO (as an intermediate between CPU64 and Intel64FSG)